src/analysis/powerset-lattice-impl.h

```diff
@@ -0,0 +1,81 @@
+#ifndef wasm_analysis_lattice_impl_h
+#define wasm_analysis_lattice_impl_h
+
+#include "lattice.h"
+
+namespace wasm::analysis {
+
+template<size_t N>
+inline void BitsetPowersetLattice<N>::copy(BitsetPowersetLattice<N>& src) {
+  value.reset();
+  value |= src.value;
```

Comment:
I just thought it might do this in-place instead of re-instantiating a bitset. But you're right, I probably should have done the simple thing, as there probably isn't any real difference between them.

Reply:
If `value = src.value` works, then do we even need this function? the default `=` operation on `BitsetPowersetLattice<N>` might be fine.

Reply:
Changed to the `=` operation.